src/modules/Accordion/Accordion.js

```diff
@@ -63,14 +63,16 @@ export default class Accordion extends Component {
     /**
      * Create simple accordion panels from an array of { text: <string>, content: <custom> } objects.
      * Object can optionally define an `active` key to open/close the panel.
+     * Object can opitonally define a `key` key used for title and content nodes' keys.
      * Mutually exclusive with children.
      * TODO: AccordionPanel should be a sub-component
      */
     panels: customPropTypes.every([
       customPropTypes.disallow(['children']),
       PropTypes.arrayOf(PropTypes.shape({
+        key: PropTypes.string,
         active: PropTypes.bool,
-        title: PropTypes.string,
+        title: customPropTypes.contentShorthand,
```

Comment:
If this prop is changed to shorthand, which we'd love, then its value should be passed to:

```jsx
AccordionTitle.create(title)
```

The `create()` method needs to be added to the title and content components to support this.  You can reference other simple components with shorthand prop for subcomponents and children, such us, the Header.  The Card is a good example of a component that has many levels of children.